internal/pkg/util/fs/mount/mount_test.go

```diff
@@ -59,8 +59,8 @@ func TestImage(t *testing.T) {
 	if err := points.AddImage(RootfsTag, "/fake", "/", "squashfs", 0, 0, 0); err == nil {
 		t.Errorf("should have failed with 0 size limit")
 	}
-	if err := points.AddImage(RootfsTag, "/fake", "/squash", "squashfs", 0, 0, 10); err == nil {
-		t.Errorf("should have failed with destination already in list")
+	if err := points.AddImage(RootfsTag, "/fake", "/squash", "squashfs", 0, 0, 10); err != nil {
```

Comment:
This line was updated incorrectly here. This function still needs a non-nil exit. I would say this:

`if err := [...]; err == nil || (err != nil && err == mount.ErrMountExists) {...}`

Reply:
Actually, I would go as far as doing this:

```
if err := [...]; err == nil {
    t.Errorf("nil error returned, should have returned non-nil mount.ErrMountExists")
} else if err != nil && err != mount.ErrMountExists {
    t.Errorf("non-nil error should have been mount.ErrMountExists")
}
```

Reply:
okay I'll give that a try!